test/new-e2e/scenarios/ndm/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/ndm/snmp"
+)
+
+func main() {
+	envNamePtr := flag.String("name", "snmp", "environment name")
+	destroyPtr := flag.Bool("destroy", false, "[optional] should destroy the environment")
+	keyPairNamePtr := flag.String("keyPairName", "", "ssh key pair name. Should be uploaded on the cloud provider")
+	apiKeyPtr := flag.String("apiKey", "", "Datadog API key")
+	appKeyPtr := flag.String("appKey", "", "Datadog APP key")
```

Comment:
Not that these should be done as part of this PR, but should end user of the framework have to care about api keys and ssh keys?

Reply:
@vboulineau has a plan to silently upload the default user public key (`id_rsa`) and add it to the remote instance known keys. For DD keys I would like to follow `ddev` and use environment variables that could be overridden with command line args

Reply:
It's already done but currently it means you must use `~/.ssh/id_rsa` to connect.